### ISSUE TITLE
[Quick links] Add the Pub 100 genie link

### ISSUE THREAD

**Issue by XavierPaquet-Rapold** [ApplETS/Notre-Dame#1110]
**Is your feature request related to a problem? Please describe.**
Add this link to the quick links: https://www.pub100genies.ca/

**Comment by LouisPhilippeHeon** [comment]
YES PLEASE, j'aurais plus à chercher le menu si j'ai pas encore reçu leur ptit code qr!!!!

**Comment by Trit0** [comment]
Mon dieu c'est vraiment une bonne idée
